src/app/stats/components/HntInfo.tsx

```diff
@@ -28,7 +27,7 @@ export const HntInfo = async () => {
       fetcher(COINGECKO_HNT_URL),
       fetchMint(HNT_MINT),
       fetchHntGovernanceStats(),
-      fetchSubDaoEpochInfo(MOBILE_MINT),
+      fetchSubDaoEpochInfo("mobile"),
```

Comment:
In order to properly cache, the values need to be primatives. So instead of passing PublicKeys we're now passing strings which are then matched up to their appropriate mint in the function itself.